lib/rom/rails/model/params.rb

```diff
@@ -53,9 +53,15 @@ def self.model_name
           RUBY
         end
 
-        def timestamps
-          attribute :created_at, DateTime, default: proc { DateTime.now }
-          attribute :updated_at, DateTime, default: proc { DateTime.now }
+        def timestamps(*attrs)
+          if attrs.empty?
+            attribute :created_at, DateTime, default: proc { DateTime.now }
+            attribute :updated_at, DateTime, default: proc { DateTime.now }
+          else
+            attrs.each do |attr|
+              attribute attr, DateTime, default: proc { DateTime.now }
+            end
+          end
```

Comment:
Do you think we could define a const like `DEFAULT_TIMESTAMPS = [:created_at, :updated_at].freeze` and use it here to reduce duplication?